tls/tls-full/config_template.yaml

```diff
@@ -0,0 +1,261 @@
+                connectAddr: "{{ default .Env.POSTGRES_SEEDS "" }}:{{ default .Env.DB_PORT "5432" }}"
+                connectProtocol: "tcp"
+                user: {{ default .Env.POSTGRES_USER "" }}
+                password: {{ default .Env.POSTGRES_PWD "" }}
+                maxConns: {{ default .Env.SQL_VIS_MAX_CONNS "20" }}
+                maxIdleConns: {{ default .Env.SQL_VIS_MAX_IDLE_CONNS "20" }}
+                maxConnLifetime: {{ default .Env.SQL_VIS_MAX_CONN_TIME "1h" }}
+        {{- end }}
+        {{- if eq $es "true" }}
+        es-visibility:
+            elasticsearch:
+                url:
+                    scheme: {{ default .Env.ES_SCHEME "http" }}
+                    host: "{{ default .Env.ES_SEEDS "" }}:{{ default .Env.ES_PORT "9200" }}"
+                username: {{ default .Env.ES_USER "" }}
+                password: {{ default .Env.ES_PWD "" }}
+                indices:
+                    visibility: {{ default .Env.ES_VIS_INDEX "temporal-visibility-dev" }} 
+        {{- end }}
+
+global:
+    membership:
+        maxJoinDuration: 30s
+        broadcastAddress: {{ default .Env.TEMPORAL_BROADCAST_ADDRESS "" }}
+    tls:
+        internode:
+            server:
+                requireClientAuth: true
+                certFile: /etc/temporal/config/certs/cluster/internode/cluster-internode.pem
+                keyFile: /etc/temporal/config/certs/cluster/internode/cluster-internode.key
+                requireClientAuth: true
+                clientCaFiles:
+                    - /etc/temporal/config/certs/cluster/ca/server-intermediate-ca-chain.pem
+            client:
+                serverName: internode.cluster-x.contoso.com
+                rootCaFiles:
+                    - /etc/temporal/config/certs/cluster/ca/server-intermediate-ca-chain.pem
+        frontend:
+            server:
+                requireClientAuth: true
+                certFile: /etc/temporal/config/certs/cluster/internode/cluster-internode.pem
+                keyFile: /etc/temporal/config/certs/cluster/internode/cluster-internode.key
+                clientCaFiles:
+                    - /etc/temporal/config/certs/cluster/ca/server-intermediate-ca-chain.pem
+            client:
+                serverName: internode.cluster-x.contoso.com
+                rootCaFiles:
+                    - /etc/temporal/config/certs/cluster/ca/server-intermediate-ca-chain.pem
+            hostOverrides:
+                accounting.cluster-x.hq.contoso.com:
+                    certFile: /etc/temporal/config/certs/cluster/accounting/cluster-accounting-chain.pem
+                    keyFile: /etc/temporal/config/certs/cluster/accounting/cluster-accounting-chain.key
+                    requireClientAuth: true
+                    clientCaFiles:
+                        - /etc/temporal/config/certs/client/ca/client-intermediate-ca-chain.pem
+            hostOverrides:
```

Comment:
Both accounting and development should be array entries under a single instance of hostOverrides